packages/legacy/core/App/defaultConfiguration.ts

```diff
@@ -55,4 +55,6 @@ export const defaultConfiguration: ConfigurationContext = {
   supportedLanguages: Object.keys(translationResources) as Locales[],
   showPreface: false,
   disableOnboardingSkip: false,
+  showScanHelp: false,
```

Comment:
do you think you could make these true by default so that this isn't a breaking change?